support/src/main/java/org/jenkinsci/plugins/workflow/support/steps/stash/StashStep.java

```diff
@@ -72,6 +74,14 @@ public String getExcludes() {
         this.excludes = Util.fixEmpty(excludes);
     }
 
+    public boolean getUseDefaultExcludes() {
```

Comment:
`isUseDefaultExcludes` is a little more conventional but I guess it does not matter.